pkgs/applications/networking/sync/unison/default.nix

```diff
@@ -26,9 +26,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   strictDeps = true;
 
-  nativeBuildInputs = [ glib wrapGAppsHook ocamlPackages.ocaml ]
-    ++ lib.optional enableX11 copyDesktopItems;
-  buildInputs = [ gsettings-desktop-schemas ncurses zlib ]
+  nativeBuildInputs = [ ocamlPackages.ocaml ]
+    ++ lib.optional enableX11 [ copyDesktopItems glib wrapGAppsHook ];
```

Comment:
```suggestion
    ++ lib.optionals enableX11 [ copyDesktopItems glib wrapGAppsHook ];
```

Reply:
Upon closer inspection, unison doesn't require glib and build&works fine without it. Is this correct?

Reply:
Inded, just tried it out and can confirm that `glib` is unnecessary.